api/app/controllers/concerns/spree/api/v2/storefront/order_concern.rb

```diff
@@ -26,7 +26,7 @@ def spree_current_order
           end
 
           def find_spree_current_order
-            Spree::Orders::FindCurrent.new.execute(
+            Spree::Api::Dependencies.storefront_current_order_finder.constantize.new.execute(
```

Comment:
Maybe it would be clearer to use `constantize` in `Spree::Api::Dependencies`?
It would reduce the amount of this calls